frontend/web/components/modals/payment/Payment.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { FC, useState, useEffect } from 'react'
+import Constants from 'common/constants'
+import InfoMessage from 'components/InfoMessage'
+import BlockedOrgInfo from 'components/BlockedOrgInfo'
+import { PricingToggle } from './PricingToggle'
+import { PricingPanel } from './PricingPanel'
+import { startupFeatures, enterpriseFeatures } from './pricingFeatures'
+import {
+  CONTACT_US_URL,
+  ON_PREMISE_HOSTING_URL,
+  SUPPORT_EMAIL,
+  SUPPORT_EMAIL_URL,
+} from './constants'
+import AccountStore from 'common/stores/account-store'
+
+type PaymentProps = {
+  viewOnly?: boolean
+  isDisableAccountText?: string
+}
+
+export const Payment = ({ isDisableAccountText, viewOnly }: PaymentProps) => {
+  const [yearly, setYearly] = useState(true)
+
+  const organisation = AccountStore.getOrganisation()
```

Comment:
Ideally we should move away from the stores. I know it might a bit out of scope but if we could have a hook (or any alternative you see) to access the organisation in TS way, that would be a really great improvement (and use it at least in those new files).

Check maybe `account-store.js` and `useOrganisation` (RTK) to see how it's done:
```
  selectOrganisation: (id) => {
    API.setCookie('organisation', `${id}`)
    store.organisation = _.find(store.model.organisations, { id })
    store.changed()
  },
```